connector/connect/src/main/scala/org/apache/spark/sql/connect/planner/SparkConnectPlanner.scala

```diff
@@ -45,12 +46,18 @@ import org.apache.spark.util.Utils
 final case class InvalidPlanInput(
     private val message: String = "",
     private val cause: Throwable = None.orNull)
-    extends Exception(message, cause)
+    extends SparkException(
```

Comment:
Can we have test for the exception ??

If it's already exists, let's convert the existing test to use `checkError` to check error class and parameters generated properly.

Reply:
done